views/components/error-components.njk

```diff
@@ -0,0 +1,8 @@
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% macro displayErrorSummary(errorList) %}
+    {{ govukErrorSummary({
+        titleText: i18n.validationErrors.errorSummary,
+        errorList: errorList
+    }) }}
+{% endmacro %}
```

Comment:
Do we need this component? Can we just use the govukErrorSummary? We are not adding any logic here

Reply:
You're right I'll remove on next commit